packages/uxpin-merge-cli/src/steps/serialization/component/implementation/typescript/component/getComponentName.ts

```diff
@@ -1,8 +1,14 @@
 import { TSSerializationContext } from '../context/getSerializationContext';
 import { getComponentFileName } from './getComponentFileName';
 import { ComponentDeclaration } from './getPropsTypeAndDefaultProps';
+import * as ts from 'typescript'
```

Comment:
Would you mind importing just `isArrowFunction`, `ArrowFunction`, and `Identifier` here?

```suggestion
import { ArrowFunction, isArrowFunction, Identifier } from 'typescript';
```